torch_ort_inference/torch_ort_inference/ortinferencemodule/__init__.py

```diff
@@ -0,0 +1,85 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import os
+import sys
+import warnings
+from cmath import e
+
+import torch
+from packaging import version
+
+from onnxruntime import set_seed
+from onnxruntime.capi import build_and_package_info as ort_info
+
+
+def _defined_from_envvar(name, default_value, warn=True):
+    new_value = os.getenv(name, None)
+    if new_value is None:
+        return default_value
+    try:
+        new_value = type(default_value)(new_value)
+    except (TypeError, ValueError) as e:
+        if warn:
+            warnings.warn("Unable to overwrite constant %r due to %r." % (name, e))
+        return default_value
+    return new_value
+
+
+################################################################################
+# All global constant goes here, before ORTInferenceModule is imported ##################
+# NOTE: To *change* values in runtime, import onnxruntime.training.ortmodule and
+# assign them new values. Importing them directly do not propagate changes.
+################################################################################
+ONNX_OPSET_VERSION = 14
```

Comment:
since this is for inference, you should just use the latest supported opset version by exporter + ort inference

Reply:
Noted. PR updated.